.circleci/config.yml

```diff
@@ -139,3 +167,26 @@ workflows:
           requires:
             - approve
             - release
+      - build_cdn:
```

Comment:
I'm keeping `build_server` step and the code related to it until everything is tested and ready to be switched over from the old deployment strategy to the new.